### PR TITLE
BULK VALIDATION description-missing

### PR DESCRIPTION
**Pull request by mSarts**
## Summary

Fixes 30 issues across 30 files.

Fixes no issue numbers.

### CODE REVIEW

**Comment by mSarts** [comment]
**Global effort to fix validation errors**

 @adegeo   

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

File: dotnet-desktop-guide/framework/winforms/advanced/windows-forms-and-unmanaged-applications.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Unmanaged apps
+description: This article provides an overview of Windows Forms an unmanaged applications, which can interoperate with some caveats.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/windows-forms-and-unmanaged-applications.md:3]:
```suggestion
description: This article provides an overview of Windows Forms and unmanaged applications, which can interoperate with some caveats.
```

File: dotnet-desktop-guide/framework/winforms/advanced/working-with-images-bitmaps-icons-and-metafiles.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Working with Images, Bitmaps, Icons, and Metafiles"
+description: Learn more about working with images, bitmaps, icons, and metafiles in this article. Learn more about working with raster and vector images.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/working-with-images-bitmaps-icons-and-metafiles.md:3]:
```suggestion
description: Learn about which articles you can read to learn more about working with images, bitmaps, icons, and metafiles.
```

Comment (mSarts) [reply dotnet-desktop-guide/framework/winforms/advanced/working-with-images-bitmaps-icons-and-metafiles.md]: Addressed.

**Review by adegeo — APPROVED** [review]

**Pull request merged**